templates/anndata_conversion.ipynb

```diff
@@ -0,0 +1,207 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `AnnData` Conversion\n",
+    "\n",
+    "The purpose of this notebook is to convert the cell table to a [`AnnData`](https://anndata.readthedocs.io/en/latest/index.html) Object.\n",
+    "\n",
+    "`AnnData` stands for Annotated Data, and is a data structure well suited for single cell data. It is a multi-faceted object composed of matrices and DataFrames which can be used to efficiently store and interact with our data.\n",
+    "\n",
+    "The following is a representation of the `AnnData` object schema:\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "  <img width=\"50%\" src=\"../docs/_images/Anndata%20Schema.svg\" alt=\"AnnData Schema\"/>\n",
```

Comment:
Can you see the image on your end? Seems like it's not showing up https://github.com/angelolab/ark-analysis/blob/anndata-conversion-fov0/templates/anndata_conversion.ipynb

Reply:
Oh, forgot to change the link to the png image.